File: mylib-system-base-manager/mylib-system-base-manager-logic-jpa/src/main/java/com/zy/mylib/system/manager/impl/CodeMapManagerImpl.java
package com.zy.mylib.system.manager.impl;

import com.zy.mylib.system.dao.CodeMapDao;
import com.zy.mylib.system.entity.CodeMap;
import com.zy.mylib.system.manager.CodeMapManager;
import com.zy.mylib.data.jpa.BaseJpaManager;
import com.zy.mylib.data.jpa.JpaRepository;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

/**
 * @author 扬
 * @date 2017/3/21
 */
@Service
public class CodeMapManagerImpl extends BaseJpaManager<CodeMap, String> implements CodeMapManager {
  @Autowired
  private CodeMapDao dao;

  @Override
  protected JpaRepository<CodeMap, String> getRepository() {
    return dao;
  }
}

File: mylib-modules/mylib-config-module/src/main/kotlin/com/zy/mylib/sys/config/manager/impl/CodeMapManagerImpl.kt
package com.zy.mylib.sys.config.manager.impl

import com.zy.mylib.sys.config.dao.CodeMapDao
import com.zy.mylib.sys.config.entity.CodeMap
import com.zy.mylib.sys.config.manager.CodeMapManager
import com.zy.mylib.mongo.manager.impl.BaseMongoManagerImpl

import javax.inject.Named

/**
 * 代码集 管理类
 * @author 代码生成器
 */
@Named
open class CodeMapManagerImpl: BaseMongoManagerImpl<CodeMapDao, CodeMap, String>(), CodeMapManager {
}